app/controllers/predicates/AuthPredicate.scala

```diff
@@ -107,27 +107,27 @@ class AuthPredicate @Inject()(authService: EnrolmentsAuthService,
       case Right(details) =>
         (details.isInsolventWithoutAccess, details.insolvencyDateFutureUserBlocked(dateService.now())) match {
           case (true, futureDateBlock) =>
-            logger.debug("[AuthPredicate][insolvencySubscriptionCall] - User is insolvent and not continuing to trade")
+            debug("[AuthPredicate][insolvencySubscriptionCall] - User is insolvent and not continuing to trade")
             Future.successful(
               Forbidden(userInsolventError()(user, request2Messages, appConfig)).addingToSession(
                 SessionKeys.insolventWithoutAccessKey -> "true",
                 SessionKeys.futureInsolvencyBlock -> s"$futureDateBlock")
             )
           case (_, true) =>
-            logger.debug("[AuthPredicate][insolvencySubscriptionCall] - User has a future insolvencyDate, throwing ISE")
+            errorLog("[AuthPredicate][insolvencySubscriptionCall] - User has a future insolvencyDate, throwing ISE")
             Future.successful(errorHandler.showInternalServerError.addingToSession(
               SessionKeys.insolventWithoutAccessKey -> "false",
               SessionKeys.futureInsolvencyBlock -> "true")
             )
           case _ =>
-            logger.debug("[AuthPredicate][insolvencySubscriptionCall] - Authenticated as principle")
+            errorLog("[AuthPredicate][insolvencySubscriptionCall] - Authenticated as principle")
             block(user).map(result => result.addingToSession(
               SessionKeys.insolventWithoutAccessKey -> "false",
               SessionKeys.futureInsolvencyBlock -> "false")
             )
         }
       case _ =>
-        logger.warn("[AuthPredicate][insolvencySubscriptionCall] - Failure obtaining insolvency status from Customer Info API")
```

Comment:
same here

Reply:
i think this should be kept as errorLog since it exits the journey